test/mocha-utils.ts

```diff
@@ -277,3 +278,18 @@ export const mochaHooks = {
     sinon.restore();
   },
 };
+
+export const expectCookieEquals = (cookies, expectedCookies) => {
+  const { isChrome } = getTestState();
+  if (!isChrome) {
+    for (const expectedCookie of expectedCookies) {
```

Comment:
What do you think about changing this to check to only checking the standard properties instead of removing the non standard ones?

Reply:
Thanks for the review! Tried to keep it simple and just copied over all the standard/shared values to a new object. Let me know if you had something else in mind.